arclightning_backend/Cargo.lock

```diff
@@ -2,6 +2,7 @@
 name = "arclightning_backend"
 version = "0.1.0"
 dependencies = [
+ "failure 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
```
cargo build
git add Cargo.lock
git commit -m "Updated cargo lock"
git push
```

just to update this, since `failure` is mentioned, but not in Cargo.toml